Modules/IO/GDCM/src/itkGDCMImageIO.cxx

```diff
@@ -301,6 +302,21 @@ GDCMImageIO::Read(void * pointer)
     image = ialut.GetOutput();
     len *= 3;
   }
+  else if (pi == gdcm::PhotometricInterpretation::YBR_FULL || pi == gdcm::PhotometricInterpretation::YBR_FULL_422 ||
+           pi == gdcm::PhotometricInterpretation::YBR_PARTIAL_420 ||
+           pi == gdcm::PhotometricInterpretation::YBR_PARTIAL_422)
```

Comment:
I don't believe PARTIAL is handled correctly but gdcm::ImageChangePhotometricInterpretation. There are also many other PhotometricInterpretation that need to be looked into, at minimal if the image provided by GDCM is not RGB a warning should be printed, that the color space/Photometric Interpretation is not supported.

Reply:
I didn't see PARTIALs for any related transfer syntax and don't have examples, so didn't add them in 1st PR. May be such files exist, who knows, and they will be handled by decoder correct, difficult to say without samples.

S. valid photometric/transfer systax combinations from current Dicom standard.

[RLE](http://dicom.nema.org/medical/dicom/current/output/chtml/part05/sect_8.2.2.html#table_8.2.2-1)
[Jpeg2000](http://dicom.nema.org/medical/dicom/current/output/chtml/part05/sect_8.2.4.html#table_8.2.4-1)
[JpegLS](http://dicom.nema.org/medical/dicom/current/output/chtml/part05/sect_8.2.3.html#table_8.2.3-1)
[Lossy Jpeg](http://dicom.nema.org/medical/dicom/current/output/chtml/part05/sect_8.2.html#table_8.2.1-1)
[Lossless Jpeg](http://dicom.nema.org/medical/dicom/current/output/chtml/part05/sect_8.2.html#table_8.2.1-2)

YBR_PARTIAL_420 is valid for MPEG files.

Also the above tables mention many extremely rare are or hypothetical combinations, eg Jpeg2000 YBR_FULL. In real life almost always there are lossy Jpeg/YBR_FULL_422 and RLE/YBR_FULL.